jstests/replsets/all_commands_downgrading_to_upgraded.js

```diff
@@ -553,6 +554,7 @@ const allCommands = {
             assert.commandWorked(conn.getDB(dbName).runCommand({drop: collName}));
         },
     },
+    createBackup: {skip: "already tested by 'backup' tests"},
```

Comment:
The goal of this test is to check that commands work in the downgrading FCV state.
"Already tested ..." message is misleading here because all commands are tested in more than one place.
So I suggest to use another message (hinted by those commands which are skipped with "requires additional authentication setup" message):
```
"requires additional setup to execute backup"
```

Reply:
Changed